test.sh

```diff
@@ -15,10 +15,15 @@ for dir in ${VERSIONS}; do
   # Kept also IMAGE_NAME as some tests might still use that.
   export IMAGE_NAME=$(docker inspect -f "{{.Config.Labels.name}}" $IMAGE_ID)
 
-  if [ -n "${TEST_MODE}" ]; then
-    VERSION=$dir test/run
-  fi
+  for test_script in ${TEST_SUITES}; do
+    if [ "${TEST_MODE}" == "container" ]; then
+      docker run --net=host -e VERSION=$dir -e IMAGE_NAME --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v /var/run/docker.sock:/var/run/docker.sock -v ${PWD}:/src -w /src docker.io/modularitycontainers/conu $test_script
```

Comment:
Would you mind to split this into multiple lines, so it can be more human-readble?

Reply:
updated